prog/vm/github_runner.rb

```diff
@@ -393,6 +393,15 @@ def setup_info
           Clog.emit("Failed to move serial.log or running journalctl") { github_runner }
         end
       end
+
+      # For analyzing the Docker Hub rate limit, we check the quota and log the
+      # remaining limit.
+      docker_quota_limit_command = <<~COMMAND
+        TOKEN=$(curl "https://auth.docker.io/token?service=registry.docker.io&scope=repository:ratelimitpreview/test:pull" | jq -r .token)
+        curl --head -H "Authorization: Bearer $TOKEN" https://registry-1.docker.io/v2/ratelimitpreview/test/manifests/latest | grep ratelimit
+      COMMAND
+      vm.sshable.cmd(docker_quota_limit_command)
```

Comment:
Let's `rescue Sshable::SshError` so that if the VM is down or something goes wrong, it doesn't block the destroy process.